dbt/models/iasworld/docs.md

```diff
@@ -0,0 +1,39 @@
+{% docs addn %}
+temp addn docs
+{% enddocs %}
```

Comment:
**[Thought, non-blocking]** Per your note in the PR body:

> I added the docs.md because the schema.yaml has the description field with a call to doc(). Let me know if I should keep that or not.

It's not strictly necessary for us to add temp docs like this -- you'll notice if you navigate to [the docs site](https://ccao-data.github.io/data-architecture/#!/source_list/iasworld) that resources without a `description` attribute get assigned a default that says "This is not currently documented" -- but I think it's fine to leave these placeholders in, we can flesh them out when we get around to https://github.com/ccao-data/data-architecture/issues/26.

![Screenshot 2023-08-24 100504](https://github.com/ccao-data/data-architecture/assets/14170650/34937c65-fdb0-4654-8c80-c45d0acd9f65)